Paccy10/ampersand_app_api
# Feature(stations): create station

**What does this PR do?**

- Create station

**Description of Task to be completed?**

- Register a new station

**How should this be manually tested?**

- Checkout to branch `ft-create-station`
- Run the app with `flask run`
- Send a `POST` request to `/stations` with the required payload
- A new station record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
